`spirv_shader_passthrough` must enable `wgpu/spirv`

## Bevy version

Main branch.

## What you did

Built `bevy` with just the `spirv_shader_passthrough` feature. (`x11` or `wayland` is required on Linux, so I included `x11` while testing.)

```bash
cargo b -p bevy --no-default-features -F spirv_shader_passthrough,x11
```

## What went wrong

```bash
error[E0599]: no variant or associated item named `SpirV` found for enum `wgpu::ShaderSource` in the current scope
  --> crates/bevy_render/src/renderer/render_device.rs:53:33
   |
53 |             wgpu::ShaderSource::SpirV(source)
   |                                 ^^^^^ variant or associated item not found in `ShaderSource<'_>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `bevy_render` (lib) due to 1 previous error
```

## Additional information

This is because the [`bevy_render/Cargo.toml`](https://github.com/bevyengine/bevy/blob/bd20382a4aec049cb78d08742a03e18836f38362/crates/bevy_render/Cargo.toml#L31) does not make the `spirv_shader_passthrough` feature depend on `wgpu`'s `wgpu/spirv` feature.

You may want to enable `bevy_render`'s `shader_format_spirv` feature, since it adds `naga` support for `spirv` as well. I don't know enough about rendering to know if this is required or not, though.

---

Caught by [`flag-frenzy`](https://github.com/TheBevyFlock/flag-frenzy).

--- a/crates/bevy_render/Cargo.toml
+++ b/crates/bevy_render/Cargo.toml
@@ -22,7 +22,7 @@ shader_format_glsl = ["naga/glsl-in", "naga/wgsl-out", "naga_oil/glsl"]
 shader_format_spirv = ["wgpu/spirv", "naga/spv-in", "naga/spv-out"]
 
 # Enable SPIR-V shader passthrough
-spirv_shader_passthrough = ["shader_format_spirv"]
+spirv_shader_passthrough = ["wgpu/spirv"]
 
 trace = ["profiling"]
 tracing-tracy = []

--- a/crates/bevy_render/Cargo.toml
+++ b/crates/bevy_render/Cargo.toml
@@ -22,7 +22,7 @@ shader_format_glsl = ["naga/glsl-in", "naga/wgsl-out", "naga_oil/glsl"]
 shader_format_spirv = ["wgpu/spirv", "naga/spv-in", "naga/spv-out"]
 
 # Enable SPIR-V shader passthrough
-spirv_shader_passthrough = []
+spirv_shader_passthrough = ["shader_format_spirv"]
 
 trace = ["profiling"]
 tracing-tracy = []

